### PR TITLE
Add Events and Performances Model (1)

### PR DESCRIPTION
**Pull request by lachiemurray**
Adds models from "Events" (races, meets, competitions) and "Performances" (individual performances / results).

Interestingly in PowerOf10 events or "Meetings" as they are referred to can span multiple days (e.g. the Olympics) so I had to move the date field from Event to Performance

### CODE REVIEW

File: phx/events/models.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import logging
+
+from django.contrib.auth.models import User
+from django.db import models
+
+logger = logging.getLogger(__name__)
+
+
+class Event(models.Model):
```
Comment (lachiemurray) [inline phx/events/models.py:9]: On reflection perhaps Events and Performances should be grouped under Results :thinking:

Comment (lachiemurray) [reply phx/events/models.py:9]: Moved everything around in this PR: https://github.com/orangespaceman/phx/pull/89

File: phx/events/admin.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class EventsAdmin(admin.ModelAdmin):
+    list_display = ['name', 'location']
+    exclude = ['uploaded_by']
+
+    def save_model(self, request, obj, form, change):
+        if getattr(obj, 'uploaded_by', None) is None:
+            obj.uploaded_by = request.user
+        obj.save()
```
Comment (orangespaceman) [inline phx/events/admin.py:15]: I'm not sure if we need this, although it's fine to leave it in. It was added initially so we could keep track of who (manually) added a news story or page, but if events (etc) are going to be added via a cron it'll usually be empty.

File: phx/events/models.py (hunk 1 above)
Comment (orangespaceman) [reply phx/events/models.py:9]: Yes, agreed. It's all related to Results so makes sense to keep them together 👍

File: phx/events/tests/models/test_events_model.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from django.test import TestCase
+
+
+class TestEventsModel(TestCase):
+
+    def test_placeholder(self):
+        self.assertEqual(1, 1)
```
Comment (orangespaceman) [inline phx/events/tests/models/test_events_model.py:7]: My favourite kind of test 🙂 

(Fine to leave in but also no need to add one if we can't think of anything useful to test...)

**Review by orangespaceman — APPROVED** [review]

File: phx/events/tests/models/test_events_model.py (hunk 3 above)
Comment (lachiemurray) [reply phx/events/tests/models/test_events_model.py:7]: I think I add some later :)

**Pull request merged**